### DISCUSSION TITLE
OsmAnd data is behind Open Street Map

### DISCUSSION THREAD

**Discussion by archer46** [osmandapp/OsmAnd#14520] [Q&A]
Hello, I've noticed that many places that have somewhat recently been added to OSM aren't showing up in OsmAnd. I get the impression that the OsmAnd is a few weeks behind OSM in terms of updates. I tested this across a few devices, and I'm getting the exact same results. How does the OsmAnd update schedule work? When would you expect things to become up to date? Thanks.

**Comment by njohnston** [comment]
Standard maps are updated monthly. For example, the most recent map files are currently from 1st June. However, updated map files aren't available on the first day of the month. It takes a while to generate them. Generally for me (UK), updated maps are available around 13/14th of the month. There's also OsmAnd Live (available through a subscription, or for free if you have enough recent OSM edits) which supports hourly updates. See the [OsmAnd map update documentation](https://docs.osmand.net/docs/user/personal/maps/#osmand-live) for details.
